src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -59,6 +60,64 @@
 
 static mtev_log_stream_t ssldb;
 
+#define MAX_DHPARAMS 8
+static struct dhparams_t {
+  int bits;
+  const char *file;
+  DH *params;
+  DH *(*builtin)();
+} dhparams[MAX_DHPARAMS] = {
+  { .bits = 2048, .builtin = DH_get_2048_224 },
+};
+static int ndhparams = 1;
+
+
+static const char *
+internal_SSL_error(int err, char *scratch, int len) {
+  switch(err) {
+#ifdef SSL_ERROR_NONE
+    case SSL_ERROR_NONE: return "SSL_ERROR_NONE";
+#endif
+#ifdef SSL_ERROR_SSL
+    case SSL_ERROR_SSL: return "SSL_ERROR_SSL";
+#endif
+#ifdef SSL_ERROR_WANT_READ
+    case SSL_ERROR_WANT_READ: return "SSL_ERROR_WANT_READ";
+#endif
+#ifdef SSL_ERROR_WANT_WRITE
+    case SSL_ERROR_WANT_WRITE: return "SSL_ERROR_WANT_WRITE";
+#endif
+#ifdef SSL_ERROR_WANT_X509_LOOKUP
+    case SSL_ERROR_WANT_X509_LOOKUP: return "SSL_ERROR_WANT_X509_LOOKUP";
+#endif
+#ifdef SSL_ERROR_SYSCALL
+    case SSL_ERROR_SYSCALL: return "SSL_ERROR_SYSCALL";
+#endif
+#ifdef SSL_ERROR_ZERO_RETURN
+    case SSL_ERROR_ZERO_RETURN: return "SSL_ERROR_ZERO_RETURN";
+#endif
+#ifdef SSL_ERROR_WANT_CONNECT
+    case SSL_ERROR_WANT_CONNECT: return "SSL_ERROR_WANT_CONNECT";
+#endif
+#ifdef SSL_ERROR_WANT_ACCEPT
+    case SSL_ERROR_WANT_ACCEPT: return "SSL_ERROR_WANT_ACCEPT";
+#endif
+#ifdef SSL_ERROR_WANT_ASYNC
+    case SSL_ERROR_WANT_ASYNC: return "SSL_ERROR_WANT_ASYNC";
+#endif
+#ifdef SSL_ERROR_WANT_ASYNC_JOB
+    case SSL_ERROR_WANT_ASYNC_JOB: return "SSL_ERROR_WANT_ASYNC_JOB";
+#endif
+#ifdef SSL_ERROR_WANT_CLIENT_HELLO_CB
+    case SSL_ERROR_WANT_CLIENT_HELLO_CB: return "SSL_ERROR_WANT_CLIENT_HELLO_CB";
+#endif
+    default:
+      if(scratch) snprintf(scratch, len, "%d", err);
```

Comment:
```suggestion
      if(scratch) snprintf(scratch, len, "unknown SSL_ERROR %d", err);
```

Reply:
In an effort to make the messages brief, this will make it say something like `ssl error: unkown SSL_ERROR %d`.. which isn't any better than `ssl error: %d`

Reply:
ah I didn't see the "ssl error:" prefix